charts/seed-bootstrap/charts/elastic-kibana-curator/templates/elasticsearch/configmap.yaml

```diff
@@ -66,6 +66,36 @@ data:
               flood_stage: ${DISK_WATERMARK_FLOOD_STAGE:95%}
           # for single node only allow only primary shard. We don't need replica
           enable: ${SHARD_REBALANCING_FOR:all}
+{{- if .Values.searchguard.enabled }}
+    # Search Guard
+    searchguard:
+      enterprise_modules_enabled: false
+      restapi:
+        roles_enabled: ["sg_all_access"]
+      authcz:
+        admin_dn:
+          - "CN=elasticsearch-logging"
+      nodes_dn:
+        - "CN=elasticsearch"
+      allow_unsafe_democertificates: true
+      #allow_default_init_sgindex: false
```

Comment:
You could remove both of allow_unsafe_democertificates and allow_default_init_sgindex - their default values are false. Currently searchguard-helm have a postStart hook that deletes the demo certificates:

```
lifecycle:
  postStart:
    exec:
      command:
      - sh
      - -c
      - |
        #!/usr/bin/env bash +e
        shopt -s dotglob
        rm -f /usr/share/elasticsearch/config/*.pem
```

We could adopt it.

https://github.com/floragunncom/search-guard-helm/blob/master/sg-helm/templates/_helpers.tpl#L29

https://docs.search-guard.com/latest/demo-installer-generated-artefacts.html#allow-demo-certificates-and-auto-initialization

Reply:
Nice, thank you :)